### DISCUSSION TITLE
No link to the repo on tags

### DISCUSSION THREAD

**Discussion by SharakPL** [community/community#14038] [Mobile] [answered]
When you tap an issue or PR you get links to organization and repo on top:

![Screenshot_20220401-131849-543.png](https://user-images.githubusercontent.com/5007456/161253808-bd5f1adc-aea5-46c4-ab6d-5b4971218f62.png)

Unfortunately it doesn't work on **tags/releases**.

**Comment by aconsuegra** [comment, marked as answer]
Hi @SharakPL, thanks for your feedback! We recently fixed this bug and it will be available in our next release (version 1.60)

**Reply by SharakPL** [reply]
Thank you :)
